Botanix/botanix-testnet-starter/project.ts

```diff
@@ -0,0 +1,89 @@
+import {
+  EthereumProject,
+  EthereumDatasourceKind,
+  EthereumHandlerKind,
+} from "@subql/types-ethereum";
+
+// Can expand the Datasource processor types via the generic param
+const project: EthereumProject = {
+  specVersion: "1.0.0",
+  version: "0.0.1",
+  name: "botanix-starter",
```

Comment:
@bgodlin script didn't quite work. In this case it should be `botanix-testnet-starter`
I note that the directory was named correctly. Should't this title and the directory name be the same?